internal/server/singleprocess/state/project.go

```diff
@@ -350,8 +350,9 @@ func (s *State) projectDelete(
 // projectIndexSet writes an index record for a single project.
 func (s *State) projectIndexSet(txn *memdb.Txn, id []byte, value *pb.Project) error {
 	record := &projectIndexRecord{
-		Id:   string(id),
-		Poll: false, // being explicit that we want to default poll to false
+		Id:       string(id),
+		Poll:     false, // being explicit that we want to default poll to false
+		ApplPoll: true,  // application polling off by default until a deployment or release happens
```

Comment:
Probably want to use `AppPoll` (not `Appl`). `Appl` is a fine shortening of application but we don't use it anywhere else while we use App prodigiously.

Reply:
👍🏻 makes sense! I used `Appl` in a bunch of places in this func so I'll fix them all up